_includes/documentation-guides-band.html

```diff
@@ -4,8 +4,9 @@
       <h3>Infinispan Guides</h3>
       <p>Get answers and find help in the Infinispan documentation set.</p>
     </div>
-    <div class="width-4-12 width-12-12-m">
-      <h4><i class="fas fa-wrench"></i> Development (Infinispan 12.0)</h4>
+    <!-- Comment the div for Development if it is not applicable. -->
+    <!--<div class="width-4-12 width-12-12-m">
```

Comment:
I've noticed we have some "if" condition like this: https://github.com/infinispan/infinispan.github.io/blob/develop/_includes/documentation-archive-band.html#L21

have you tried with something like this instead of comments?
```
{% if site.data.ispn.unstable %}
div...
{% endif %}
```

Reply:
@pruivo I don't know what we'd do without you